mockgcp/mockmanagedkafka/normalize.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mockmanagedkafka
+
+import (
+	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockgcpregistry"
+)
+
+var _ mockgcpregistry.SupportsNormalization = &MockService{}
+
+const NormalizedTimestamp = "2024-04-01T12:34:56.123456Z"
+
+func (s *MockService) ConfigureVisitor(url string, replacements mockgcpregistry.NormalizingVisitor) {
```

Comment:
A random question that’s unrelated to this PR: do the `ConfigureVisitor` functions from different services override each other?

Reply:
They do, currently .... it might be interesting to scope them only to the "matching" hosts, but I'm not aware of any problems (?) and it is nice to get some normalizations "for free"